### ISSUE TITLE
Adding labels to own addresses from Accounts page

### ISSUE THREAD

**Issue by keerifox** [Nault/Nault#124]
At the moment it's very easy to miss out on the information that you can label your own addresses, as you have to visit the address book page to see the hint.

Accounts should have a pencil icon next to them with hover text "Change Label" for quick access to creation of a new address book contact, with the address prefilled. One small issue is there is no functionality for changing labels, only removing/creating new.

**Comment by aleksre** [comment]
Like this?

![image](https://user-images.githubusercontent.com/1097444/89343131-1d227700-d6a4-11ea-879d-068c9e9e936e.png)

Personally I think it would be great to add the same pencil icon and inline label edit to the account card as well. I also really think the default name (Account#0 etc.) should be used consistently across the app.

**Comment by keerifox** [comment]
nevermind, it is a duplicate of #111 then, this UI is quite a hide and seek..

yeah it's worth displaying Account #number when no label is set

i don't want to overload the account card with actions and information but once #127 is merged it will lead to this page anyway where you can edit it

**Comment by Joohansson** [comment]
Yeah, I'm not really a fan of these copy and edit buttons only is visible on hover throughout the wallet. Easy to miss and don't work on mobile. I vote for making all buttons always visible but need to test this and see how it feels. I can take a look and make a PR.
